src/meeting/meet/speakersObserver.ts

```
@@ -0,0 +1,675 @@
+                                    participantCohortId =
+                                        cohortElement.getAttribute('data-cohort-id')
+                                }
+
+                                // Add this participant to the matching merged group
+                                if (
+                                    participantCohortId &&
+                                    mergedGroups.has(participantCohortId)
+                                ) {
+                                    mergedGroups
+                                        .get(participantCohortId)!
+                                        .members.push(ariaLabel)
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)